CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.8.0
+* Updated Rokt SDKs and interfaces
```

Comment:
Maybe it is good to add "Introducing BottomSheet layout"?

Reply:
I feel it should be good to put in our rokt.docs and not in pub.dev.

Reply:
Yeah but for the new features, I think it would be more professional to put it in the pub.dev like what we did for version 3.7.0